### PR TITLE
Event API Routes

### PR DESCRIPTION
**Pull request by tausifmuzaffar**
W-4006595

### CODE REVIEW

File: api/v1/helpers/verbs/findUtils.js [hunk 1]
```diff
@@ -152,6 +152,10 @@ function toSequelizeWhere(filter, props) {
           const v = filter[key][j];
           if (typeof v === 'boolean') {
             values.push(v);
+          } else if (typeof v === 'number') {
```
Comment (iamigo) [inline api/v1/helpers/verbs/findUtils.js]: Good!

File: api/v1/swagger.yaml [hunk 2]
```diff
@@ -6941,6 +6941,148 @@ paths:
...
+        -
+          name: queryBody
+          in: body
+          required: true
+          schema:
+            type: object
+            description: Create event properties
+            properties:
+              log:
+                type: string
```
Comment (iamigo) [inline api/v1/swagger.yaml]: adding descriptions to each of these params will help api users

File: api/v1/swagger.yaml [hunk 3]
```diff
@@ -6941,6 +6941,148 @@ paths:
           $ref: "#/responses/genericError"
 
 # =============================================================================
+  /events:
+    x-swagger-router-controller: events
+    get:
+      summary: Retrieve all events
+      tags: [ events ]
```
Comment (iamigo) [inline api/v1/swagger.yaml]: pls. add "description" attribute (a more verbose version of the "summary" attribute) to all of these new endpoints

**Comment by tausifmuzaffar** [comment]
@annyhe do you mind looking at this review while Ian is on PTO?

**Comment by tausifmuzaffar** [comment]
@pallavi2209 could you look at this PR?

**Comment by tausifmuzaffar** [comment]
@iamigo @pallavi2209 are there more changes needed here?

**Comment by iamigo** [comment]
I had comments about adding description attributes. If you promise to add them in a subsequent PR, I'm prepared to approve this PR.

**Comment by tausifmuzaffar** [comment]
@iamigo I had added descriptions to the routes and the post parameters, did you get a chance to see them? Just now i also added description to the response.

**Pull request merged**